src/avt/Database/Database/CMakeLists.txt

```diff
@@ -7,28 +7,41 @@
 #
 #****************************************************************************
 
-set(DATABASE_SOURCES
-    avtDatabase.C
-    avtDatabaseFactory.C
-    avtDatasetCollection.C
-    avtDatasetDatabase.C
-    avtFileDescriptorManager.C
-    avtGenericDatabase.C
-    avtIOInformation.C
-    avtPolygonToTrianglesTesselator.C
-    avtPolyhedralSplit.C
-    avtSILGenerator.C
-    avtSourceFromDatabase.C
-    avtTransformManager.C
-    avtVariableCache.C)
+visit_append_list(
+    NAME  avtdatabase_ser_SOURCES
+    ITEMS ${CMAKE_CURRENT_SOURCE_DIR}/avtDatabase.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatabaseFactory.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatasetCollection.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatasetDatabase.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtFileDescriptorManager.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtGenericDatabase.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtIOInformation.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtPolygonToTrianglesTesselator.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtPolyhedralSplit.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtSILGenerator.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtSourceFromDatabase.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtTransformManager.C
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtVariableCache.C)
 
-target_sources(avtdatabase_ser PRIVATE ${DATABASE_SOURCES})
-target_include_directories(avtdatabase_ser PUBLIC .)
+visit_append_list(
+    NAME  avtdatabase_ser_HEADERS
+    ITEMS ${CMAKE_CURRENT_SOURCE_DIR}/avtDatabase.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatabaseFactory.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatasetCollection.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtDatasetDatabase.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtFileDescriptorManager.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtGenericDatabase.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtIOInformation.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtPolygonToTrianglesTesselator.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtPolyhedralSplit.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtSILGenerator.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtSourceFromDatabase.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtTransformManager.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/avtVariableCache.h
+          ${CMAKE_CURRENT_SOURCE_DIR}/database_exports.h)
 
-if(VISIT_PARALLEL)
-    target_sources(avtdatabase_par PRIVATE ${DATABASE_SOURCES})
-    target_include_directories(avtdatabase_par PUBLIC .)
-endif()
-
-unset(DATABASE_SOURCES)
+visit_append_list(
+    NAME  avtdatabase_ser_INCLUDES
+    ITEMS $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+          $<INSTALL_INTERFACE:${VISIT_INSTALLED_VERSION_INCLUDE}/visit/avt/Database/Database>)
```

Comment:
I see all the work here to `append_list` for the `_ser` lists. But, there was logic to also handle the `_par` targer sources and  include directories for `if(VISIT_PARALLEL)` here in this file. Where has that moved to now?

Reply:
In this case, the root CMakeLists.txt (avt/Database/CMakeLists.txt ) copies the _ser_XXX vars to _par_XXX vars.
The reason being: visit_add_library deletes variables *possibly* created by visit_append_list.
After visit_add_library call, the variables would not be available for the parallel target if they weren't copied.
The deletion is cued off of the target name for ease of cleanup, hence the use of _ser in the var name.